### PR TITLE
Update automation-levels.md

### PR DESCRIPTION
**Pull request by amitkumart**
As per the finding on ICM https://portal.microsofticm.com/imp/v3/incidents/details/406318273/home For SMB users, the automation level is not taken from the device groups but rather from the endpoint themselves through TVM, the value that is checked is RTP level: Realtime protection state (Enabled or Disabled).

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/automation-levels.md [hunk 1]
```diff
@@ -59,6 +59,8 @@ Automated investigation and remediation (AIR) capabilities in Microsoft Defender
 - If your security team has defined device groups with a level of automation, those settings aren't changed by the new default settings that are rolling out.
 
 - You can keep your default automation settings, or change them according to your organizational needs. To change your settings, [set your level of automation](/microsoft-365/security/defender-endpoint/configure-automated-investigations-remediation#set-up-device-groups).
+- > [!NOTE]
+  > Defender for Business depents on realtime protection for Automatic invetigation , so Defender for Business should be running in active mode to have the Automatic invetigation
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/automation-levels.md]:
```suggestion

> [!NOTE]
> Defender for Business depends on real-time protection for automatic investigation, so it should be running in active mode to enable automatic investigation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @amitkumart, kindly commit the suggested changes. Many thanks.

File: microsoft-365/security/defender-endpoint/automation-levels.md (hunk 1 above)
Comment (amitkumart) [reply microsoft-365/security/defender-endpoint/automation-levels.md]: Changes look good to me, Thanks

**Review by denisebmsft — APPROVED** [review]
Thank you @amitkumart @JohanFreelancer9

**Pull request merged**